docs/stencil-docs/reference-docs/stencil-utils-api-reference.md

```diff
@@ -52,13 +51,34 @@ updateView() {
     this.refreshView(content);
   });
 }
- ```
+```
+
+The Cornerstone theme uses the `getPage` method in the following contexts:
 
-### Usage in Cornerstone
 - [`assets/js/theme/wishlist.js`](https://github.com/bigcommerce/cornerstone/blob/master/assets/js/theme/wishlist.js)
 - [`assets/js/theme/gift-certificate.js`](https://github.com/bigcommerce/cornerstone/blob/master/assets/js/theme/gift-certificate.js)
 - [`assets/js/theme/faceted-search.js`](https://github.com/bigcommerce/cornerstone/blob/master/assets/js/theme/common/faceted-search.js)
 
+### Get page by GraphQL
+
+[getPageByGQL](https://github.com/bigcommerce/stencil-utils/blob/master/src/api/index.js)
+
+This method takes the form `utils.api.getPageByGQL(page, callback)`. 
+
+It allows you to render parts of a template using an inline GraphQL query, as shown in [Front Matter Reference - GraphQL attributes](/stencil-docs/reference-docs/front-matter-reference#graphql-attributes).
+
+| Parameter | Type | Description/Usage |
+|:----------|:-----|:------------------|
+| page | string | Presentation template; for example, `pages/store-locator`. <br> The `template` option allows you to select a particular template, or an array of templates, to render one page. Each value must correspond to a file in the theme's `templates/pages/` subdirectory. |
```

Comment:
this array of templates thing is confusing, since you said this is a string. if page can also take an array, revise type and provide an additional example.

Reply:
It looks like there might be verbiage you can adapt starting at line 425.

Reply:
I added array as a type and copyedited the description. 

@bc-apostoliuk Could you provide an example of an array of templates for the page parameter?

Reply:
@bc-andreadao that's my mistake. The template parameter accepts the single template. So it means we need to fix the description to: `The 'template' option allows you to select a particular template to render one page. The value must correspond to a file in the theme's `templates/pages/` subdirectory.`

Reply:
Got it, thanks @bc-apostoliuk